cmd/url.go

```diff
@@ -83,19 +123,45 @@ var urlDeleteCmd = &cobra.Command{
 
 		// Initialization
 		client := getOcClient()
+		projectName := project.GetCurrent(client)
 		applicationName, err := application.GetCurrent(client)
 		checkError(err, "")
+
+		var componentName string
+
+		if urlComponent == "" {
+			componentName = getComponent(client, "", applicationName, projectName)
```

Comment:
the same comments as in above, you can remove this whole if statement, just by providing correct second argument to `getComponent`